IntelliNode/wrappers/OpenAIWrapper.js

```diff
@@ -5,9 +5,12 @@ Copyright 2023 Github.com/Barqawiz/IntelliNode
 
    Licensed under the Apache License, Version 2.0 (the "License");
 */
+
 const axios = require('axios');
 const ProxyHelper = require('../utils/ProxyHelper');
 const connHelper = require('../utils/ConnHelper');
+const fs = require('fs');
+const https = require('https');
```

Comment:
The code already using Axios for HTTP connection, let's not use extra libs for this